### PR TITLE
Fix: weights should be float64

### PR DESCRIPTION
**Pull request by till**
Fixes the following error:

```
json: cannot unmarshal number 100.5 into Go struct field WeightGoal.goal.startWeight of type int64
```

### CODE REVIEW

**Comment by till** [comment]
Added a second commit for reading the nutrition log. If you prefer 2 PRs and have any feedback, lmk. :)

File: types/nutrition.go [hunk 1]
```diff
@@ -0,0 +1,47 @@
+package types
+
+type FoodLog struct {
+	IsFavorite bool       `json:"isFavorite"`
+	LogDate    FitbitDate `json:"logDate"`
+	LogId      int64      `json:"logId"`
+	LoggedFood struct {
```
Comment (galeone) [inline types/nutrition.go]: Can you remove the nested structs and declare the types outside?
e.g. A type `LoggedFood` and a type `Nutritional` with an optional (`json:"-"`) value for the `Water` field.

This way you can re-use the same data type for FoodLog.LoggedFoor and for FoodLogList.Summary

**Comment by galeone** [comment]
Hey thank you for this PR! I just left a comment that I'd like you to address. Thanks for spotting the type issue though

**Comment by till** [comment]
@galeone I don't see the comment yet, but yeah... let me know. :)

**Review by galeone — CHANGES_REQUESTED** [review]

**Comment by galeone** [comment]
> @galeone I don't see the comment yet, but yeah... let me know. :)

O forgot to submit lol

File: types/nutrition.go (hunk 1 above)
Comment (till) [reply types/nutrition.go]: Updated, though I think `"-"` would ignore it everywhere, I did an `omitempty`. And that seems to work. :)

**Review by galeone — APPROVED** [review]

**Comment by galeone** [comment]
Merged, thanks! You can `go get` version 2.1.0 containing your changes

**Pull request merged**